### PR TITLE
Merge master into feature-rds

### PR DESCRIPTION
**Pull request by markdboyd**
Notable change: carries the change from #2048 to use Terraform v0.13.6 into the `feature-rds` branch

### CODE REVIEW

File: docs/tasks.md [hunk 1]
```diff
@@ -154,7 +163,7 @@ Updates CMR metadata to set access constraints
 
 ### [@cumulus/update-granules-cmr-metadata-file-links](https://github.com/nasa/cumulus/tree/master/tasks/update-granules-cmr-metadata-file-links)
 
-Update CMR metadata files with correct online access urls and etags and transfer etag info to granules' CMR files
+Update cmr metadata files with correct online access urls and etags and transfer etag info to granules' CMR files
```
Comment (botanical) [inline docs/tasks.md:166]: I know you didn't change this but I am curious why this is lowercase now

File: tf-modules/cumulus-rds-tf/main.tf [hunk 2]
```diff
@@ -1,6 +1,9 @@
 terraform {
   required_providers {
-    aws  = ">= 3.5.0"
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.5.0"
```
Comment (botanical) [inline tf-modules/cumulus-rds-tf/main.tf:5]:
```suggestion
      version = ">= 3.14.1"
```

`main/master` has `3.14.1

**Review by botanical — CHANGES_REQUESTED** [review]
Had one question about a version difference

File: docs/tasks.md (hunk 1 above)
Comment (markdboyd) [reply docs/tasks.md:166]: This is auto-generated from the package.json for the task:

https://github.com/nasa/cumulus/blob/feature-rds/tasks/update-granules-cmr-metadata-file-links/package.json#L4

So this got changed when I re-generated the docs, as I did locally when making some docs. I can update the package.json and re-generate though

Comment (markdboyd) [reply docs/tasks.md]: https://github.com/nasa/cumulus/pull/2053/commits/7b1cfdab46b68976d2eff97c4015ac43d80000c9

File: tf-modules/cumulus-rds-tf/main.tf (hunk 2 above)
Comment (markdboyd) [reply tf-modules/cumulus-rds-tf/main.tf]: https://github.com/nasa/cumulus/pull/2053/commits/8f3fd99ce7d1bdbbccbc77adba636c02731ca057

**Review by botanical — APPROVED** [review]
Thanks for addressing my feedback! Looks good to me.

**Pull request merged**